### DISCUSSION TITLE
how to use OneTable Migrate and the Migrate CLI

### DISCUSSION THREAD

**Discussion by ismaeil-shajar** [sensedeep/dynamodb-onetable#434] [Q&A] [answered]
in [your article](https://www.sensedeep.com/blog/posts/2021/onetable-cli.html) you tag some samples for OneTable migrate but now look like you deleted them, where I can find some examples ?

**Comment by mobsense** [comment, marked as answer]
Sorry, I think that sample is dead. I'll remove the link.

**Reply by ismaeil-shajar** [reply]
Ok can we get another valid one ?

**Reply by dev-embedthis** [reply]
Anyone want to contribute an example?
